webrender/src/renderer.rs

```diff
@@ -1787,33 +1787,60 @@ impl Renderer {
         let debug_flags = options.debug_flags;
         let payload_tx_for_backend = payload_tx.clone();
         let recorder = options.recorder;
-        let worker_config = ThreadPoolConfig::new()
```

Comment:
The worker_config is only used when we don't have the thread pool. So, move the config into the worker's unwrap_or_else() block.